src/selectors/app.js

```diff
@@ -164,3 +171,17 @@ export const getTimelineHeight: Selector<null | CssPixels> = createSelector(
     return height;
   }
 );
+
+export const getIsNewProfileLoadAllowed: Selector<boolean> = createSelector(
```

Comment:
Selectors are also really good places for code documentation, as they are the public interface into the data. It would probably be good to explain this mechanism a little bit.